engine/orchestration/src/main/java/org/apache/cloudstack/engine/orchestration/NetworkOrchestrator.java

```diff
@@ -563,6 +569,11 @@ public void doInTransactionWithoutResult(final TransactionStatus status) {
                 }
 
                 _networkOfferingDao.persistDefaultL2NetworkOfferings();
+
+                final String mtu_size =_configDao.getValue(KVM_MTU_SIZE);
+                if (nonNull(mtu_size)) {
```

Comment:
Can we use our utils for this or otherwise use Apache Commons notNull?

Reply:
Hi, sure, but what's wrong with Java 1.7 java.util.Objects?

Reply:
As we mainly use those tools in CloudStack for validating

Reply:
changed, please check again